MAINTAINERS.md

```diff
@@ -0,0 +1,69 @@
+# Maintainers
+
+## The maintainer list of all repositories(names are in alphabetical order)
+* Chunlin Yang (Redhat, @clyang82)
+* Dong Beiqing (eBay, @dongbeiqing)
+* Hao Lin (eBay, @haowells)
+* Jian Qiu (Redhat, @qiujian16)
+* Jian Zhu (Redhat, @zhujian7)
+* Le Yang (Redhat, @elgnay)
+* LongLong Cao (Redhat, @morvencao)
+* Mike Ng (Redhat, @mikeshng)
+* Qing Hao (Redhat, @haoqing0110)
+* Zhao Xue (Redhat, @xuezhaojun)
+* Zhiwei Yin (Redhat, @zhiweiyin318)
+* Wei Liu (Redhat, @skeeey)
+* David Eads (Redhat, @deads2k)
+
+## The maintainers in different areas of OCM
+
+### [OCM CORE](https://github.com/open-cluster-management-io/ocm)
+* General maintainers:
+    * Jian Qiu
+* Maintainers for specific areas:
+    * [api](https://github.com/open-cluster-management-io/api)
+        * Jian Qiu
+        * David Eads
+    * `registration`
+        * Le Yang
+        * Wei Liu
+        * Zhao Xue
+    * `work`
+        * Le Yang
+        * Jian Zhu
+    * `placement`
+        * Le Yang
+        * Qing Hao
+    * `operator`
+        * Jian Zhu
+        * Zhiwei Yin
+        * Dong Beiqing
+        * Hao Lin
+
+### SDK and Tools
+* [addon-framework](https://github.com/open-cluster-management-io/addon-framework)
+    * Jian Qiu
+    * Le Yang
+    * Zhiwei Yin
+    * Jian Zhu
+* [sdk-go](https://github.com/open-cluster-management-io/sdk-go/blob/main/OWNERS)
+    * Jian Qiu
+    * David Eads
+* [clusteradm](https://github.com/open-cluster-management-io/clusteradm)
+    * Jian Qiu
+    * Zhu Jian
+
+### Integrations and Addons
+* [multicluster-controlplane](https://github.com/open-cluster-management-io/multicluster-controlplane)
```

Comment:
How about add a note here to clearify the different level of maintainibility? @qiujian16 

```markdown
Note: We welcome all community contributors to develop their addons in or out of the ocm-io and link them here.
The maintainers of integrations and addons only need to be responsible for their own repos.
```

Reply:
yes, I think it is ok.

Reply:
The Integrations and Addons part is removed, added a "Note" to describe, and also link the "Integrations and Addons" section of website.